### PR TITLE
Implement inline svg

### PR DESCRIPTION
**Pull request by thekp**
Adds an inline `SVG` element,  enclosed within an `a` tag, to the header of the article. Also adds accessibility attributes to the SVG.

* [x] Fixes https://github.com/bbc/simorgh/issues/368
* [x] Tests added for new features

* [ ] Test engineer approval

### CODE REVIEW

File: src/app/components/Header/index.jsx
Comment (radiocontrolled) [inline src/app/components/Header/index.jsx]: The required accessibility attributes, `focusable="false"` and  `aria-hidden="true"` documented in https://github.com/bbc/simorgh/issues/368 are not present on the SVG.

File: src/app/containers/Article/__snapshots__/index.test.jsx.snap
Comment (radiocontrolled) [inline src/app/containers/Article/__snapshots__/index.test.jsx.snap]: I can see you have added the visually hidden text and it ends up being rendered as a hidden `<span>`.  The `(this is hidden text)` should be removed however.

File: src/app/components/Header/index.jsx
Comment (radiocontrolled) [inline src/app/components/Header/index.jsx]: The SVG requires further styling so it can be centred vertically.
screen shot 2018-07-27 at 16 48 17

**Review by radiocontrolled — CHANGES_REQUESTED** [review]
I have left a few comments regarding visual style and  accessibility attributes required in this issue.

**Comment by radiocontrolled** [comment]
As @thekp is back working on WS stuff next week, I'll pick up where he leaves off with this PR next week 👍

File: src/app/components/Header/index.jsx
Comment (sareh) [reply src/app/components/Header/index.jsx]: This has been added.

**Review by radiocontrolled — DISMISSED** [review]

File: src/app/components/Header/__snapshots__/index.test.jsx.snap
Comment (sareh) [inline src/app/components/Header/__snapshots__/index.test.jsx.snap]: By adding the style in the `Header/index.jsx` file via the `svgStyle` object, in the output here you get the object rather than key value pairs.  It will likely be better to add the attributes directly with the other SVG attributes like viewBox.

Comment (radiocontrolled) [reply src/app/components/Header/__snapshots__/index.test.jsx.snap]: I thought I might get that in a review :) - will update now.

File: cypress/integration/articleSpec.js
Comment (jtart) [inline cypress/integration/articleSpec.js]: Same comments as https://github.com/bbc/simorgh/pull/387/files#r206078930. This test should be moved to their own explicit test and file. Please work closely with @jamesbrumpton on adding cypress tests.

**Review by jtart — CHANGES_REQUESTED** [review]

File: src/app/components/Header/__snapshots__/index.test.jsx.snap
Comment (radiocontrolled) [reply src/app/components/Header/__snapshots__/index.test.jsx.snap]: I've reverted this change so this ticket isn't adding style and is focusing instead on the semantics the inline SVG should accomplish. 

The styling will be addressed in https://github.com/bbc/simorgh/issues/346

Therefore my comment earlier about the SVG needing to be styled can be ignored.

**Comment by greenc05** [comment]
Have you looked at this with some assistive technology @radiocontrolled and in High Contrast Mode on PC?

**Comment by radiocontrolled** [comment]
@greenc05 not yet, but it will do after the second code review - it will go into the UX & Accessibility Review column.

**Comment by greenc05** [comment]
@radiocontrolled You should really look at it with AT before code review incase you need to make any changes...

File: src/app/components/Header/index.jsx
Comment (jtart) [inline src/app/components/Header/index.jsx]: Can this be named `OffscreenText`?

File: cypress/integration/articleSpec.js
Comment (jtart) [inline cypress/integration/articleSpec.js]: I think there would be value in wrapping this in a describe block that is something like logo, so it is clear that we are testing a11y of the logo, not just the header.

**Review by jtart — CHANGES_REQUESTED** [review]

File: src/app/components/Header/index.jsx
Comment (greenc05) [reply src/app/components/Header/index.jsx]: or visuallyHiddenText?

**Review by PaulChinkwende — DISMISSED** [review]

**Review by jtart — DISMISSED** [review]

**Comment by greenc05** [comment]
@radiocontrolled does the Article workflow mean that PR's aren't merged until after they have passed the 'UX & Accessibility Review' column? This PR (and all PR's with front end code changes) should be looked at with some AT (and in this case with high contrast mode on PC) by a developer before they are merged.

**Comment by greenc05** [comment]
@radiocontrolled I would be interested in seeing this bug, I've already broken my sandbox so can't check this out, perhaps you can show me on Monday when I'm next in? https://github.com/bbc/simorgh/pull/389/commits/026a9fab45a799d15611f4511a8153ba09cdb38f

**Comment by radiocontrolled** [comment]
@greenc05, re your comment, yes that's the intent of the workflow 

> does the Article workflow mean that PR's aren't merged until after they have passed the 'UX & Accessibility Review' column?

**Comment by radiocontrolled** [comment]
@greenc05 yes, I can demo that on Monday and I will pause work on this until then.

I am unclear what I should be hearing from a screen reader perspective in JAWS. What I think I'm hearing is "BBC News" twice: once from the header's aria-label, and once from the visually hidden text in the span inside the link containing the SVG. 

I get the same in i0S VO (iPhone 5): it reads the visually hidden text containing "BBC News" as a visited link, then the header banner's aria-label.

**Comment by greenc05** [comment]
@radiocontrolled the headers aria-label was added in error, and the PR that did that https://github.com/bbc/simorgh/pull/392 should have been reverted by @pjlee11 ? If you merge in the latest you shouldn't have the header aria-label if all is well?

**Comment by radiocontrolled** [comment]
@greenc05 merging latest removes hearing 'BBC News' twice.

**Comment by radiocontrolled** [comment]
**Testing instructions**
* Please see Catherine's comments above regarding an AT with high contrast mode
* As described in https://github.com/bbc/simorgh/issues/368 when tabbing to the logo, you should expect to hear "BBC News" once 

Note: 
Natalie has given UX approval to this ticket. Visual styling should be addressed in https://github.com/bbc/simorgh/issues/346

**Comment by greenc05** [comment]
@thekp should the svg have an image element like we do on the News front page?
```<image src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAHoAAAAgCAMAAADqkpxHAAAAG1BMVEX////35+fvxsbnra3ejIzWa2vOUlLGMTG9GBibW0q6AAAACXRSTlP//////////wBTT3gSAAACWUlEQVR4nLyX65aFIAiFE/Hy/k88pagbpDOetWaNv6yQD2FLddV/HiVnmV33CI4F3/dJ5vEygx8XzySbVdHzBoaF2vpALPcn4xt0fbxEh2LjWZ4E3JHcjTcXJ+i875A9Zzcv8JitESo6+4gOMHjuQa+TdKp7T4TXekrEkRR6Y1v0LzVZFOsrjjzk9aikAOjwWqFX9Ca0URhlO40YgypME32RZh+gN6E9bpKJZ/l5ZgUedDTv7BO0EVrz0uIB4+cytVmw4m/ovkrJ4wRthNavdDxLZHVTZUf3E4fJO0FroTVKMfHE5ba1G8i4oIUNQR2hm9CSoTBuGyrfQguAEHQtQbOP0Epog4LCV2khc/IGuq9YQZ2hobBrAQhtiWzubjEmukuNhuHHRqq3zWaWZ1GzNhb2OEvgKuERO0SnYQaUJTQQWX/CwmaDln1/hZ6FRcoUGlRdRpaNR4OWN4+PxtcH+CLwMygjnq3Jw8ajQfcjRh7al9kUmqaI0JTIFrxvnDUajtgpWuSlW1WLJxddG7PxUGztBvsY3QzZUFokVmSGHQ1ajnc+R7e82f6cmjgu9arCQbJGZyX1bJyjx2cPSrnIvY8K2dDyFjtHZ48i8WAmcD5Cs1rgy8T8Gb31ZohHmYFFcRMO36yH6OSg6/YWvu+sy+jJDNmH6LJR6vhkMcGMN0d7mDz0OPWIxmYWzALypOyXnygyiXMXLTJ47eEmBdlLCtlOxtpHfUF3mZyib8z2C/F4sPvB/57yilYthYMZFp08D+T8TfVkU6+5t+hvht/ISpmt5wcAAP//AwCBpFE5ZTGfKQAAAABJRU5ErkJggg=="></image>```

**Comment by radiocontrolled** [comment]
@greenc05 @thekp is world service working on this on a part time basis. I will look into this.

Ideally the need for a fallback should have been specified on the original issue. https://github.com/bbc/simorgh/issues/368

**Review by PaulChinkwende — DISMISSED** [review]

**Comment by radiocontrolled** [comment]
@jimjohnsonrollings, a question has been raised about whether there should be a PNG fallback to the SVG logo. Can you please raise a separate issue for this if you want it to be added.

**Review by jtart — DISMISSED** [review]

**Comment by radiocontrolled** [comment]
The visual styling for this ticket is addressed in https://github.com/bbc/simorgh/issues/346

File: src/app/components/Article/__snapshots__/index.test.jsx.snap
Comment (greenc05) [inline src/app/components/Article/__snapshots__/index.test.jsx.snap:12]: @radiocontrolled out of interest which implementation/approach have you followed for this 'visually hidden' CSS? What we have is slightly different from what we use in Grandstand for the class 'gs-u-vh'. You mentioned webaim earlier, is this their approach?

**Pull request merged**
